docker/Dockerfile

```diff
@@ -21,9 +21,9 @@ RUN /opt/conda/bin/pip install --no-cache-dir \
     tensorflow-datasets==2.0.0 \
     jupyterlab==1.2.6 \
     boto3==1.11.13 \
-    adversarial-robustness-toolbox==1.2.0 \
-    Pillow==7.0.0 \
-    pydub==0.23.1 \
+    adversarial-robustness-toolbox==1.3.0 \
+    Pillow==7.1.2 \
+    pydub==0.24.1 \
     apache-beam==2.18.0 \
```

Comment:
Let's update the following (and also in the host-requirements.txt):
apache-beam==2.22.0
dill==0.3.1.1

Also, let's fix ffmpeg to 4.2.3 (from 4.2), as that seems to clean up the conda process a bit.